bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -154,9 +154,21 @@ spec:
               name: {{ template "mariadb-galera.fullname" . }}
             {{- end }}
               key: ldap-bindpw
+        - name: MARIADB_LDAP_NSS_INITGROUPS_IGNOREUSERS
+          value: {{ .Values.ldap.nss_initgroups_ignoreusers | default "root,nslcd" | quote }}
```

Comment:
Move these defaults to **values.yaml** (uncomment them) and remove the default command. Please note that unless `.Values.ldap.enabled` is `true`, they won't be used.